db/migrate/20180422030623_create_slack_user.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSlackUser < ActiveRecord::Migration[5.0]
+  def change
+    create_table :slack_users do |t|
+      t.string :slack_id, index: true
+      t.string :slack_name
+      t.string :slack_real_name
+      t.string :slack_display_name
+      t.string :slack_email, index: true, foreign_key: true
+    end
```

Comment:
You'll need to add in the timestamp columns.  Here is an example of that:

https://github.com/OperationCode/operationcode_backend/blob/master/db/migrate/20170920093119_create_code_schools.rb#L12